api/stories/storyNewModel.js

```diff
@@ -17,8 +17,22 @@ const add = (story) => {
  * @param {number} ID the ID to search for in the database
  * @returns {Promise} a promise that resolves to story object of the given story ID
  */
-const getById = (ID) => {
-  return db('Stories-New').where({ ID });
+const getById = async (ID) => {
+  const story = await db('Stories-New').where('Stories-New.ID', ID);
+  const episodes = await getEpisodesByStoryID(ID);
+  const episodesArray = [];
+  let i = 0;
+  for (i; i < episodes.length; i++) {
```

Comment:
Why are we choosing to scope that index var `i` above the for loop outside of the loop's parens.

Reply:
Great question! There is not a good answer. This is fixed on branch "feature/add-prompts-to-episodes" which I will PR tonight.  https://github.com/Lambda-School-Labs/scribble-stadium-be/blob/feature/add-prompts-to-story/api/stories/storyNewModel.js